_includes/head.html

```diff
@@ -31,4 +31,5 @@
   </style>
 <![endif]-->
 
-<meta http-equiv="cleartype" content="on">
\ No newline at end of file
+<meta http-equiv="cleartype" content="on">
+<script async defer src="{{ '/assets/js/main.min.js' | absolute_url }}"></script>
```

Comment:
Moves script into head and loads it asynchronously. This allows browsers to start the download as soon as they can (much sooner than before the close of the body) but do so in a non-blocking manner. Big benefits here for hosts supporting HTTP/2, and in some cases modern browsers may perform optimistic preloading as described in the Google analytics doc I linked.